src/service/receipt.service.ts

```diff
@@ -29,8 +32,24 @@ import {
 } from 'nem2-sdk';
 
 export class ReceiptService {
+    private transactionStatementsTable: HorizontalTable;
```

Comment:
I'd create one class per transaction table, so we can test each table in the future.

Example: https://github.com/nemtech/nem2-cli/blob/master/src/commands/block/header.ts#L36